examples/systemcalls/systemcalls.c

```diff
@@ -11,12 +19,17 @@ bool do_system(const char *cmd)
 {
 
 /*
- * TODO  add your code here
  *  Call the system() function with the command set in the cmd
  *   and return a boolean true if the system() call completed with success
  *   or false() if it returned a failure
 */
 
+    int res = system(cmd);
+    if(res == -1)
```

Comment:
Is -1 the correct check here?  I think there could be other return values which could also be considered a failure.